exporters/otlp/src/otlp_grpc_client.cc

```diff
@@ -58,17 +68,20 @@ std::shared_ptr<grpc::Channel> OtlpGrpcClient::MakeChannel(const OtlpGrpcExporte
   grpc::ChannelArguments grpc_arguments;
   grpc_arguments.SetUserAgentPrefix(options.user_agent);
 
-  if (options.use_ssl_credentials)
-  {
+  if (options.use_ssl_credentials) {
     grpc::SslCredentialsOptions ssl_opts;
-    if (options.ssl_credentials_cacert_path.empty())
-    {
-      ssl_opts.pem_root_certs = options.ssl_credentials_cacert_as_string;
-    }
-    else
-    {
-      ssl_opts.pem_root_certs = GetFileContents((options.ssl_credentials_cacert_path).c_str());
-    }
+    ssl_opts.pem_root_certs = GetFileContentsOrInMemoryContents(
+        options.ssl_credentials_cacert_path,
+        options.ssl_credentials_cacert_as_string);
+#if ENABLE_OTLP_GRPC_SSL_MTLS_PREVIEW
+    ssl_opts.pem_private_key = GetFileContentsOrInMemoryContents(
+      options.ssl_client_key_path,
+      options.ssl_client_key_string);
+    ssl_opts.pem_cert_chain = GetFileContentsOrInMemoryContents(
+      options.ssl_client_cert_path,
+      options.ssl_client_cert_string);
+#endif
+    };
```

Comment:
It seems this line breaks the build.

Please make sure the code builds locally with grpc before pushing, it will save a few round trips.

Reply:
Apologies for that, I tested a later patch, but not exactly this - missing it.